src/pushsource/_impl/backend/errata_source/errata_source.py

```diff
@@ -68,6 +71,14 @@ def __init__(
                 removed when no longer needed. Only use this if you know that you
                 need it.
 
+            keytab_path (str):
+                Kerberos keytab path for authenticating with Errata HTTP API.
+                Only necessary for container pushes.
```

Comment:
I would suggest leaving out the "Only necessary for container pushes" text as it's likely for this to be changed imminently.  Aidan is currently working on replacing the XML-RPC client with usage of the HTTP API and that'll make these arguments have an effect on all errata.

Also, it's technically not necessary even for container pushes since you made it support env vars too.

Reply:
Removed the line.